scantool/diag.h

```diff
@@ -301,21 +301,24 @@ int diag_flcalloc(const char *name, const int line,
 //diag_flmalloc : do not call directly !
```

Comment:
Oops, I forgot to remove those!

Reply:
Shall I make a force push or ...?

Reply:
yep since this is your feature branch, you can rebase + force push as you like. Let me know when you're done !

Reply:
Done.